crates/trie/parallel/src/proof.rs

```diff
@@ -278,20 +276,27 @@ where
                                 ParallelStateRootError::StorageRoot(StorageRootError::Database(
                                     DatabaseError::Other(e.to_string()),
                                 ))
+                            })?;
+
+                            raw_fallback_proof.try_into().map_err(|e_rlp: alloy_rlp::Error| {
+                                ParallelStateRootError::Other(format!(
+                                    "Failed to decode fallback storage proof for {hashed_address}: {e_rlp}",
```

Comment:
can we `.try_into()?` here instead of the custom err message?

Reply:
Yeah, done!